cfgov/unprocessed/js/modules/util/data-hook.js

```diff
@@ -60,8 +60,22 @@ function contains( element, value ) {
   return values.indexOf( value ) > -1 ? true : false;
 }
 
+/**
+ * @param {string} behavior
+ *  Behavior type used to find the element within the dom.
+ * @returns {HTMLNode} if it exists in the dom, null otherwise.
+ */
+function get( behavior ) {
+  var behaviorSelector =
+  standardType.JS_HOOK + '*=' + standardType.BEHAVIOR_PREFIX + behavior;
+  var behaviorElement = document.querySelector( '[' + behaviorSelector + ']' );
```

Comment:
So I'm going to remove this function and use [behavior.find](https://github.com/cfpb/cfgov-refresh/blob/1e8a673b248ca58fa55b8e9cfd9804f9f7f433a9/cfgov/unprocessed/js/modules/util/behavior.js#L127) instead.